src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -44,16 +43,6 @@ class SideNav extends Component {
 
     // Escape early if the item has children.
     if (hasChildren) {
-      // Flip the item's expanded property.
-      this.setState({
-        navItemsLookup: {
-          ...navItemsLookup,
-          [id]: {
-            ...navItemsLookup[id],
-            expanded: !expanded,
-          },
-        },
-      });
       recordEvent({ event: 'nav-sidenav' });
```

Comment:
I think I'd just move this `recordEvent` directly into `NavItemRow.js` since the actual event is the same every time

Reply:
The last case is a bit different https://github.com/department-of-veterans-affairs/vets-website/blob/d7bd356443766d069644f044d14a082d9e20a646/src/platform/site-wide/side-nav/components/SideNav.js#L68  and there is couple of checks needed before tracking the event- would be nice to have input from @kelsonic or @ethanteague

Reply:
Makes no difference to me 🙂 Both work so I'd advocate for whatever you think will be easier to maintain long-term!

Reply:
I believe the only difference is [this line](https://github.com/department-of-veterans-affairs/vets-website/pull/13199/files/07fef3a3ed2e946a43b50c711138c11343e6dee4#diff-aba68ce4f937195d55cd9399052709c4R57) that @brangi pointed out, so just keep in mind the events are different for nav items with and without children and we should be good! 🙂